test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpDirectivesTest.cs

```diff
@@ -12,29 +12,16 @@ namespace Microsoft.AspNet.Razor.Test.Parser.CSharp
     public class CSharpDirectivesTest : CsHtmlCodeParserTestBase
     {
         [Fact]
-        public void TagHelperPrefixDirective_NoValueSucceeds()
-        {
-            ParseBlockTest("@tagHelperPrefix \"\"",
-                new DirectiveBlock(
-                    Factory.CodeTransition(),
-                    Factory
-                        .MetaCode(SyntaxConstants.CSharp.TagHelperPrefixKeyword + " ")
-                        .Accepts(AcceptedCharacters.None),
-                    Factory.Code("\"\"")
-                        .AsTagHelperPrefixDirective(string.Empty)));
-        }
-
-        [Fact]
         public void TagHelperPrefixDirective_Succeeds()
         {
-            ParseBlockTest("@tagHelperPrefix \"Foo\"",
```

Comment:
Have you preserved tests showing what happens when parsing a `.cshtml` file containing the old syntax? Where are those tests? And what will the end user see in that case, either in console logging or on the error page (assuming that's enabled)?